android/src/main/java/com/swmansion/rnscreens/Screen.kt

```diff
@@ -371,6 +372,20 @@ class Screen(
         }
     }
 
+    private fun isInsideScrollViewWithRemoveClippedSubviews(child: ViewGroup): Boolean {
```

Comment:
It would be nicer for this method to live in `ViewExt.kt` for example. I don't like this code in `Screen` implementation.

Reply:
https://github.com/software-mansion/react-native-screens/pull/2383/commits/c1e59b14253c64ffc65c0d628ed5f9c17359fdad